testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -302,19 +301,4 @@ public void testSelectInformationSchemaColumns()
                 .skippingTypesCheck()
                 .matches("VALUES 'regionkey', 'name', 'comment'");
     }
-
-    @Test
-    public void testShowCreateTable()
-    {
-        // SHOW CREATE TABLE exercises table properties and comments, which may be skipped during regular SELECT execution
-        assertThat((String) computeActual("SHOW CREATE TABLE region").getOnlyValue())
```

Comment:
I think it is good to have a single such test in smoke test. However the expected value here may vary depending on connector. Some connectors could print default values for some properties in `SHOW CREATE TABLE`.

Reply:
In such case, a connector may need to override the method with an explicit assertion